code/weapon/weapon.h

```diff
@@ -473,7 +473,10 @@ struct weapon_info
 	// Energy suck effect
 	float weapon_reduce;					// how much energy removed from weapons systems
 	float afterburner_reduce;			// how much energy removed from weapons systems
-
+	
+	// Vampirism Effect -Strygon 7/23/2021
+	float succ_factor;					// Factor by which a vampiric weapon will multiply the healing done to the shooter
```

Comment:
Amusing, but you should keep silliness to localized areas of code, this field will show up in autocomplete, and passers-by could get quite confused, so you shouldn't need to track down comments to figure out what this is.

Reply:
Hint, rename it to something like `vampire_factor` or `vampiric_factor`